File: src/0557-Reverse-Words-in-a-String-III/reverse_words_in_a_string_III.py
class Solution:
    def reverseWords(self, s: str) -> str:
        words = s.split()

        reversed_words = [word[::-1] for word in words]
        return ' '.join(reversed_words)

    
if __name__ == "__main__":
    s: str =  "Let's take LeetCode contest"
    solution = Solution()
    
    print(solution.reverseWords(s))

File: src/0557-Reverse-Words-in-a-String-III/reverse_words_in_a_string_III.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
  let words = s.split(' ')
  
  let reversedWords = words.map(word => {
    return word.split('').reverse().join('')
  })

  return reversedWords.join(' ')
};

const s = "Let's take LeetCode contest";

console.log(reverseWords(s))